src/integration/java/io/pinecone/helpers/TestUtilities.java

```diff
@@ -0,0 +1,71 @@
+package io.pinecone.helpers;
+
+import io.pinecone.clients.Index;
+import io.pinecone.clients.Pinecone;
+import io.pinecone.proto.DescribeIndexStatsResponse;
+import io.pinecone.proto.NamespaceSummary;
+import org.openapitools.client.model.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TestUtilities {
```

Comment:
This is a rename of `IndexManager` -> `TestUtilities`. @aulorbe had mentioned it was a bit confusing getting started with some of these helpers, and since the index management has been moved to `TestIndexResourcesManager`, we can rename this class to be a bit more informative.

Reply:
ty!